surfnperf.js

```diff
@@ -356,6 +356,26 @@
     return this.duration('navigationStart', 'loadEventEnd');
   };
 
+  /**
+   * Time to first paint (amount of time that lapses before some useful content is diplayed indicating that the browser is not stuck and is still working)
+   *
+   * @returns {integer} time in ms
+   * @memberOf SurfNPerf
+   */
+  SNPProto.getFirstPaint = function() {
+    if(window.chrome && window.chrome.loadTimes) {
+      return window.chrome.loadTimes().firstPaintTime;
+    } else if (window.msPerformance && window.msPerformance.timing && window.msPerformance.timing.msFirstPaint) {
+      return window.msPerformance.timing.msFirstPaint;
```

Comment:
should actually be window.performance.timing.msFirstPaint - window.performance.timing.navigationStart (see https://msdn.microsoft.com/en-us/ff974719)